Tools/scripts/run_coverage.py

```diff
@@ -110,28 +115,29 @@ def check_build(self, name, path):
 
     def run_build(self, use_example=False):
         """Clean the build directory and build binaries for coverage."""
-        os.environ["CCFLAGS"] = os.environ.get("CCFLAGS", "") + " -fprofile-arcs -ftest-coverage"
+        os.environ["CFLAGS"] = os.environ.get("CFLAGS", "") + " -fprofile-arcs -ftest-coverage"
```

Comment:
Am I misunderstanding your patches against waf?  Shouldn't these be unncessary?

If I were you I'd actually remove the changes to boards.py and do that move-flags-to-waf bit to a separate PR.  I'd probably not merge that sort of change without taking it to a devcall - esp. when we're trying to get 4.1s out the door.

Reply:
removed here.